dbt/include/trino/macros/materializations/incremental.sql

```diff
@@ -59,6 +61,54 @@
     )
 {%- endmacro %}
 
+{% macro trino__get_merge_sql(target, source, unique_key, dest_columns, predicates=none) -%}
+    {%- set predicates = [] if predicates is none else [] + predicates -%}
+    {%- set dest_cols_csv = get_quoted_csv(dest_columns | map(attribute="name")) -%}
+    {%- set dest_cols_csv_source = dest_cols_csv.split(', ') -%}
+    {%- set update_columns = config.get('merge_update_columns', default = dest_columns | map(attribute="quoted") | list) -%}
+    {%- set sql_header = config.get('sql_header', none) -%}
+
+    {% if unique_key %}
+        {% if unique_key is sequence and unique_key is not mapping and unique_key is not string %}
+            {% for key in unique_key %}
+                {% set this_key_match %}
+                    DBT_INTERNAL_SOURCE.{{ key }} = DBT_INTERNAL_DEST.{{ key }}
+                {% endset %}
+                {% do predicates.append(this_key_match) %}
+            {% endfor %}
+        {% else %}
+            {% set unique_key_match %}
+                DBT_INTERNAL_SOURCE.{{ unique_key }} = DBT_INTERNAL_DEST.{{ unique_key }}
+            {% endset %}
+            {% do predicates.append(unique_key_match) %}
+        {% endif %}
+    {% else %}
+        {% do predicates.append('FALSE') %}
```

Comment:
https://github.com/trinodb/trino/issues/13751#issue-1344466027

Shouldn't we do the `INSERT` instead of the `MERGE` here in `dbt-trino`?

Why should we make use of an edge of `MERGE` when we can very well benefit of `INSERT`?

Reply:
Done, now all tests are enabled by default.

Reply:
Great catch @findinpath